docs/source/getting-started/installation.rst

```diff
@@ -271,6 +279,36 @@ RunPod
   runpod config
 
 
+Fluidstack
+~~~~~~~~~~~~~~~~~~
+`Fluidstack <https://fluidstack.io/>`__ is a cloud provider offering low-cost GPUs. To configure Fluidstack access, go to the `Home <https://console.fluidstack.io/`_ page on your Fluidstack console to generate an API key and then add the :code:`API key` to :code:`~/.fluidstack/api_key` and the :code:`API token` to :code:`~/.fluidstack/api_token`.
```

Comment:
Should we add a mkdir -p and `echo ... > ...` commands here just like we have for lambda cloud above?